pkg/client/src/app/pages/applications/analysis-wizard/options.tsx

```diff
@@ -1,27 +1,116 @@
 import * as React from "react";
```

Comment:
The file name 'options' is somewhat terse. I think we should at least change to options-step or something similar to convey what it is referring to. Tough to follow what is going into this file.